.github/workflows/connector-performance-command.yml

```diff
@@ -133,15 +142,18 @@ jobs:
           DS: ${{ github.event.inputs.dataset }}
           prefix: '{"type":"LOG","log":{"level":"INFO","message":"INFO i.a.i.p.PerformanceTest(runTest):165'
           suffix: '"}}'
+          HARNESS_TYPE: ${{ steps.which-harness.outputs.harness_type }}
         run: |
           kubectl apply -f ./tools/bin/admin-service-account.yaml
           connector_name=$(echo $CONN | cut -d / -f 2)
           kind load docker-image airbyte/$connector_name:dev --name chart-testing
-          kind load docker-image airbyte/source-harness:dev --name chart-testing
+          kind load docker-image airbyte/$HARNESS_TYPE:dev --name chart-testing
           export CONNECTOR_IMAGE_NAME=${CONN/connectors/airbyte}:dev
           export DATASET=$DS
-          envsubst < ./tools/bin/source-harness-process.yaml | kubectl create -f -
-          POD=$(kubectl get pod -l app=source-harness -o jsonpath="{.items[0].metadata.name}")
+          export HARNESS=$HARNESS_TYPE
```

Comment:
what uses this HARNESS var?

Reply:
`envsubst` on the following line set the correct image name to use spinning up the harness pod in run-harness-process.yaml:
```
image: airbyte/$HARNESS:dev
```
Using $HARNESS_TYPE would have been cleaner but it is not working unless you explicitly `export`.
I couldn't really figure out why

Reply:
huh, what line is this? I can't seem to find it

Reply:
https://github.com/airbytehq/airbyte/blob/722125466d107bdec318781d67eff5a666786095/tools/bin/run-harness-process.yaml#L10
 it's the very last code change at the bottom of this PR

Reply:
oh got it! I think the reason you need to export is because the var is not saved to the BASH context.

probably useful to add a comment explaining these are used for the pod yaml. it's not that clear otherwise.

Reply:
It's as if evtsubst is running its own context. Didn't get to the bottom of it since a simple export worked.
Adding comment to explain